articles/Angle/agEUR_post_Euler.md

```diff
@@ -0,0 +1,217 @@
+- Before the hack, Angle had $36.1m (Core Module), $11.8m (SLP), and a surplus (treasury) of $5.6m. The exploit led to Angle Protocol losing approximately 17.6m USDC, leaving agEUR undercollateralized.
+- Angle's governance multisigs (Guardian and Governor, despite the relatively high signature threshold) were quick to pause the protocol's critical functions, minimizing further negative impact.
+- Although present in the documents, the principle of seniority, which prioritizes agEUR holders over the Stablecoin Liquidity Providers (SLPs) and Hedging Agents (HAs), was not clearly stated and had to be voted on shortly after the exploit.
+- The Euler incident caused considerable disruptions in agEUR associated Curve pools, resulting from a significant net outflow of euro-denominated stablecoins (>€8m).
+- Euler's successful recovery of the stolen funds led to Angle protocol restoring over-collateralization. Angle may not have survived had the exploited funds been lost, leading to significant losses for SLP and HAs and a dire collateral ratio for agEUR.
+- This de-peg event serves as a reminder of the various trade-offs associated with different stablecoin designs. While fiat-redeemable stablecoins may present challenges regarding transparency and regulatory compliance (EURS, EURT, EUROC), crypto-collateralized stablecoins like Angle grapple with capital efficiency concerns and potentially expose users to other protocol risks.
+
+## Revisiting agEUR and Prior Report Findings: Balancing Overcollateralization and Capital Efficiency
+
+The Angle Protocol presents a novel approach to decentralized finance by offering stablecoins backed by diversified collateral. Its primary product, the agEUR stablecoin, has gained recognition due to its unique method of balancing over-collateralization with capital efficiency. This balance is achieved by distributing risks among various market participants, namely: the **Core Module** (CM), **Stablecoin Liquidity Providers** (SLP), and **Hedging Agents** (HAs). The agEUR stablecoin aims to maintain its peg to the Euro by leveraging a diverse collateral pool. Overcollateralization is a fundamental aspect of this design, ensuring that agEUR remains stable even during market fluctuations.
+
+### Market Participants: Roles and Contributions
+![](https://i.imgur.com/gUR3jGm.png)
+
+#### Core Module (CM)
+The Core Module oversees collateral pools and mints stablecoins. Users can deposit collateral and obtain agEUR in return. At the same time, the Core Module sustains a debt-to-collateral ratio to ensure the stablecoin's over-collateralization.
+
+#### Stablecoin Liquidity Providers (SLPs)
+SLPs are providing liquidity that enables the exchange of agEUR with other stablecoins. In return, they receive fees generated from the trades they facilitate. SLPs are instrumental in maintaining agEUR's liquidity and contribute to the overall stability of the stablecoin.
```

Comment:
Add some context how it works to be an SLP. WHat are the counterparty assets, is liquidity generally distributed evenly or has there been a dominant pool?